torchvision/prototype/transforms/functional/_misc.py

```diff
@@ -2,10 +2,13 @@
 
 import PIL.Image
 import torch
-from torchvision.transforms import functional_tensor as _FT
+from torchvision.transforms import functional_tensor as _FT, functional_pil as _FP
 from torchvision.transforms.functional import to_tensor, to_pil_image
 
 
+get_dimensions_image_tensor = _FT.get_dimensions
+get_dimensions_image_pil = _FP.get_dimensions
```

Comment:
The new names of the kernels. We should consider moving them to `_meta_conversion` and renaming it to `_meta`. This will allow for easier inclusion without worrying for cyclic dependencies.